bindings.c

```diff
@@ -3544,7 +3604,9 @@ static int proc_meminfo_read(char *buf, size_t size, off_t offset,
 	memlimit = get_min_memlimit(cg, "memory.limit_in_bytes");
 	if (!cgfs_get_value("memory", cg, "memory.usage_in_bytes", &memusage_str))
 		goto err;
-	if (!cgfs_get_value("memory", cg, "memory.stat", &memstat_str))
+
+	mstat = (struct memory_stat *)malloc(sizeof(struct memory_stat));
```

Comment:
We don't cast `malloc()`.

Reply:
> We don't cast `malloc()`.

thanks for your comment, I have fixed it. please refer to my master branch